src/main/resources/config.yml

```diff
@@ -80,6 +80,8 @@ traits:
   zoglin-baby: true
   piglin-baby: true
   leashed: true
+  # Paper
```

Comment:
Remove this comment and follow the conventions at the beggining of the file :D ``# (**)`` comment